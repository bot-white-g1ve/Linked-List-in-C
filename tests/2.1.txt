NEW 0
INSERT 0 0 a
INSERT 0 1 b
VIEW 0
INSERT 1 0 a
VIEW ALL