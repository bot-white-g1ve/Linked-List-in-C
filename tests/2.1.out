List 0: 
List 0: a
List 0: a -> b
a -> b
INVALID COMMAND: INSERT
Number of lists: 1
List 0
